src/components/Input/index.tsx

```diff
@@ -9,9 +9,13 @@ export const OutlinedInput: FC<TextFieldProps> = ({ children, ...props }) =>
       sx: {
         fontSize: 14,
         fontWeight: 400,
-        lineHeight: 1.7,
-        px: 3,
-        py: 2
+        lineHeight: 1.7
+      },
+      InputProps: {
+        sx: {
+          py: 0.7,
+          px: 1.5
+        }
```

Comment:
if you will add sx (styles props) to TextField it will cause text field change. but TextField is wrapper for Input (and some others components), so if you want to update padding inside input borders you need to add it to InputProps

<img width="1305" alt="image" src="https://github.com/anamanoilo/react-uomo-shop/assets/15706494/c87544d2-e4cc-469e-9f2e-e61460dc97d3">

this is TextField look like